packages/primitives/button/test/fixtures/bare-bones.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { Button } from '../../src'
+
+const App = () => (
+  <Button
+    aria-label={'accessibilityLabel'}
```

Comment:
this should have failed with typescript, we do not have `aria-label` prop
we do have `accessibilityLabel` prop 😉